UnitsNet/BaseDimensions.cs

```diff
@@ -41,6 +42,18 @@ public BaseDimensions(int length, int mass, int time, int current, int temperatu
             LuminousIntensity = luminousIntensity;
         }
 
+        public bool IsBase()
+        {
+            var dimensionsArray = new int[]{Length, Mass, Time, Current, Temperature, Amount, LuminousIntensity};
+            bool onlyOneEqualsOne = dimensionsArray.Where(dimension => dimension == 1).Take(2).Count() == 1;
```

Comment:
`dimensionsArray.Count(dimension => dimension == 1) == 1`

Reply:
That would be less efficient as it must go through the entire array. This will stop once it finds 2.

It's small array though so it isn't a huge deal. Is this a readability concern?

Reply:
Ah yes I found it confusing. I would favor readability since we're talking 7 items here, but either way works for me.

Reply:
Never mind, merging this now.